Brain/Brain.ino

```diff
@@ -50,22 +59,26 @@ valveInfo valve_ids[7] = {
 valveInfo valve = valveInfo("",0);
 
 void setup() {
-  Wire.begin();       
+  Wire.begin();
   Serial.begin(9600);
   RFSerial.begin(57600);
   for (int i=0; i<sizeof(all_ids)/sizeof(sensorInfo); i++) {
     sensor_checks[i][0] = all_ids[i].clock_freq;
     sensor_checks[i][1] = 1;
   }
+
+  Ducers::init(&Wire);
+  Barometer::init(&Wire);
+  GPS::init(&GPSSerial);
 }
 
-void loop() { 
+void loop() {
   String command = "";
 //  command = RFSerial.readString();
   command = "{26,1|456}";
```

Comment:
I assume this won't be hard coded for flight, and this is just for testing?

Reply:
Yup this was just in there for testing purposes. I'll take it out.